doc/api/buffer.markdown

```diff
@@ -577,6 +577,46 @@ for (var key of buf.keys()) {
 //   5
 ```
 
+### buf.lastIndexOf(value[, byteOffset][, encoding])
+
+* `value` {String, Buffer or Number}
```

Comment:
It's done like this everywhere else in the file -- eg `indexOf`, `includes`, and `fill`

I can change all of those places if you want--LMK